option optimize_for = LITE_RUNTIME;

message CsRiverLanternOthersGetItemNotify
{
 repeated string name = 1;
 repeated uint32 item_type = 2;
 repeated uint32 item_id = 3;
 repeated uint32 main_id = 4;
}
